### PR TITLE
Add instructions for running docs locally on Windows 10

### LINKED ISSUE #1503: Bug: Verify and Update README instructions for running docs locally on Windows
## Issue Description
Issue reported on Windows 10
```
C:\Users\User\dev\pantheon\documentation>bin\sculpin generate --server

Fatal error: Call to a member function getPrettyVersion() on null in C:\Users\User\dev\pantheon\documentation\vendor\sculpin\sculpin\src\Sculpin\Bundle\SculpinBundle\Console\Application.php on line 57
```

## Suggested Resolution
Verify process for running docs locally on windows, update README as needed.

### PR DESCRIPTION
**Pull request by rachelwhitton**
Closes #1503 

## Effect
PR includes the following changes:
- Support Windows 10 and Git for Windows 

cc @katekligman 

Tested on a VM running Windows 10 and PHP 5.5, Sculpin downloaded as a project dependency produces a fatal error, as a workaround the README instructs users to install Sculpin globally and update the System PATH variable.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -81,14 +80,51 @@ Fork and clone this repository. Issue pull-requests one document at a time.
...
+
+1. If you do not already have PHP installed, use [Windows Web Platform Installer](https://www.microsoft.com/web/gallery/install.aspx?appid=PHP56) or [install manually](http://php.net/manual/en/install.windows.manual.php).
+2. Install [Git for Windows](https://git-for-windows.github.io/) using the default settings and open Git BASH.
+3. Clone the documentation repository.
+4. Install [Sculpin](https://sculpin.io) within the current users root directory (e.g. `/c/Users/username`):
+
+ ```
+ curl -O https://download.sculpin.io/sculpin.phar
+ chmod +x sculpin.phar
+ ```
```
Comment (katekligman) [inline README.md]: The above line can be removed: chmod isn't used on windows.

File: README.md [hunk 2]
```diff
@@ -81,14 +80,51 @@ Fork and clone this repository. Issue pull-requests one document at a time.
...
+ Open the Start Menu and search for System (Control Panel), click to open. Select **Advanced System Setttings** then click **Environment Variables**. Select the **PATH** variable then click **Edit** and **New**. Enter the path for the current users root directory (e.g. `C:\Users\username`) and click **Okay**. Restart your machine.
```
Comment (katekligman) [inline README.md]: They won't need to restart their machine, only the command prompt instance.

Comment (katekligman) [inline README.md]: Also, they'll need to associate .phar files with the php.exe program. Windows will prompt them to do this the first time sculpin is run.

File: README.md [hunk 3]
```diff
@@ -81,14 +80,51 @@ Fork and clone this repository. Issue pull-requests one document at a time.
...
+
+ ```
+ sculpin.phar generate --server
+ ```
+ Visit the local docs site at: <http://localhost:8000/docs>
+
+ Finally, you can tell sculpin to watch the docs directory and automatically
+ regenerate upon file modification:
+ ```
+ sculpin.phar generate --server --watch
```
Comment (katekligman) [inline README.md]: Despite adding a file association for php and adding this to my environment PATH, I could only get this to work on windows 10 with the following line:

php c:\users\kate\sculpin.phar generate --server --watch

Typing sculpin.phar would invoke it, but the other command line arguments were discarded.

**Comment by rachelwhitton** [comment]
+1 Thanks for your help @katekligman! Tested on my vm, worked like a charm. 🚢

**Pull request merged**